kong/plugins/statsd/log.lua

```diff
@@ -301,20 +303,20 @@ if ngx.config.ngx_lua_version >= 10011 then
           ["node"] = hostname,
         }
         logger:send_statsd("lmdb.used_space",
-          lmdb_info.last_used_page * lmdb_info.page_size, logger.stat_types.gauge,
+          lmdb_used_size, logger.stat_types.gauge,
           metric_config.sample_rate, tags, conf.tag_style)
         logger:send_statsd("lmdb.capacity",
-          lmdb_info.map_size, logger.stat_types.gauge,
+          lmdb_capacity, logger.stat_types.gauge,
           metric_config.sample_rate, tags, conf.tag_style)
 
       else
         local lmdb_used_space_metric_name = conf.hostname_in_prefix and "lmdb.used_space" or string_format("node.%s.lmdb.used_space", hostname)
```

Comment:
We should fix this metrics name to be consistent with the API as well.

`dbless.lmdb.in_use_bytes`
`dbless.lmdb.map_size_bytes`

Also we can expose `dbless.lmdb.max_readers` and `dbless.lmdb.num_readers` as well. More metrics is always better.